api/nakadi-event-bus-api.yaml

```diff
@@ -316,7 +316,9 @@ paths:
         incoming Events. Validation rules are evaluated in the order they are defined and the Event
         is **rejected** in the first case of failure. If the offending validation rule provides
         information about the violation it will be included in the `BatchItemResponse`.  If the
-        `EventType` defines schema validation it will be performed at this moment.
+        `EventType` defines schema validation it will be performed at this moment. The size of each
+        Event will also be validated. The maximum size per Event is 1,000,000 bytes, before
```

Comment:
@antban we had agreed to set it to something a bit lower, but I was waiting for today's standup to agree on a value. I've suggested 999,000 bytes on hipchat.